examples/tutorials/01_basics/01_hello_world.py

```diff
@@ -16,22 +16,15 @@
 
 '''Hello World
 
-Make Cozmo say 'Hello World' in this simple Cozmo SDK example program.
+Make Cozmo say 'Hello World' and then wait for him to finish speaking
```

Comment:
it was an attempt to document the .wait_for_completed(). a bit, but I'll change it back as it doesn't really help explain it.